src/Discord.Net.Rest/Entities/Channels/ChannelHelper.cs

```diff
@@ -76,15 +76,16 @@ internal static class ChannelHelper
         public static async Task<RestInviteMetadata> CreateInviteAsync(IGuildChannel channel, BaseDiscordClient client,
             int? maxAge, int? maxUses, bool isTemporary, bool isUnique, RequestOptions options)
         {
-            var args = new CreateChannelInviteParams { IsTemporary = isTemporary, IsUnique = isUnique };
-            if (maxAge.HasValue)
-                args.MaxAge = maxAge.Value;
-            else
-                args.MaxAge = 0;
-            if (maxUses.HasValue)
-                args.MaxUses = maxUses.Value;
-            else
-                args.MaxUses = 0;
+            if (maxAge.HasValue && maxAge > 86400)
```

Comment:
I'm pretty sure you don't have to check `.HasValue` since the lifting of the operator already does that.

Reply:
Just checked, it does work as expected: https://sharplab.io/#v2:D4AQDABCCMDcCwAoEBmKAmCBhCBvJEhUaIALBALIAUAlHgUYwJYB2ALgPwQAeEAvBDAJEjRg1EQmAMwhVeAPggBWGuIlEYATioAiNgFMAzmx01hjAL5ILQA=

Reply:
I'm surprised the compiler doesn't put the `HasValue` check first there. :eyes: